pyproject.toml

```diff
@@ -76,10 +68,6 @@ exclude = [
 
   [tool.poetry.extras]
   docs = [
-  "insegel",
-  "sphinx",
-  "sphinxcontrib-autoprogram",
-  "sphinxcontrib-spelling",
   "toml"
```

Comment:
The _toml_ dependency was used in the `docs/conf.py` python script used to build the documentation with _sphinx_.
It used _toml_ to parse the `pyproject.toml` and grab the project name, version, etc.

Do we still need it?